eng/Localize/LocProject.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
You may want to fix indents in this json file.

Reply:
It's auto generated - if we change it pipeline will fail.